src/dockerfile.asb.asbs

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/windows/servercore:1803
+
+WORKDIR /servicecontrol
+
+ADD /ServiceControl.Transports.ASBS/bin/Release/net461 .
```

Comment:
We should consider using an intermediate container to do the actual build but I don't think we need to worry about it just yet.